test/verify/check-machines

```diff
@@ -3435,6 +3438,73 @@ class TestMachines(MachineCase, StorageHelpers, NetworkHelpers):
         b.click('#deactivate-network-test_network4-{0}'.format(connectionName)) # Deactivate transient network
         self.waitNetworkRow("test_network4", connectionName, False) # Check it's not present after deactivation
 
+    def testSnapshots(self):
+        # Checks if difference of @time1 and @time2 is not greater than @difference (in seconds)
+        def checkTimeDiff(time1, time2, difference):
+            tmp = time2.split(' ') # split "Today 13:13:13" into day and time
+            if not tmp[1].startswith('00:') and not tmp[0] == "Today":
+                return False
+
+            diff = datetime.strptime(time1, '%H:%M:%S') - datetime.strptime(tmp[1] + tmp[2], '%I:%M%p')
+            return -difference < diff.total_seconds() < difference
+
+        b = self.browser
+        m = self.machine
+
+        self.startVm("subVmTest1")
+
+        if m.image in distrosWithoutSnapshot:
+            return
```

Comment:
Please let the test case run a little further, and make sure that the snapshots tab is *not* shown. This will ensure that the code correctly deals with a nonexisting API; although other tests implicitly cover that as well. But more importantly, it will make sure that we don't forget to update `distrosWithoutSnapshot` when an image refresh brings in a libvirt-dbus version that supports snapshots.

Reply:
Fixed